<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpdrGuiMainWindow</class>
 <widget class="QMainWindow" name="SpdrGuiMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabSynchronize">
       <attribute name="title">
        <string>Synchronize</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayoutSynchronize">
        <item row="4" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>354</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="widgetBasicSynchronize" native="true">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBoxAdvancedOptions">
          <property name="title">
           <string>Advanced options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QCheckBox" name="checkBoxRemoveEmptyDirectiories">
             <property name="toolTip">
              <string>&lt;b&gt;Default:&lt;/b&gt; checked

During the synchronization, it may happen that some folders in the output folder become empty (due to files being moved, renamed, or removed). If this ontrol is checked, they will be removed</string>
             </property>
             <property name="text">
              <string>Remove empty directories</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxRemoveMissingFiles">
             <property name="toolTip">
              <string>&lt;b&gt;Default:&lt;/b&gt; checked

It may happen that a file is found in the output folder, which has no counterpart in the input folder (Spdr also checks for renamed, moved and updated files, don't worry!). If that control is checked, those files will be removed.</string>
             </property>
             <property name="text">
              <string>Remove missing files</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="checkBoxDeepSearch">
             <property name="toolTip">
              <string>&lt;b&gt;Default:&lt;/b&gt; checked

Spdr checks whether files are identical using numerous techniques. One of them is Md5 checksums. It may happen - although it is extremely unlikely - that 2 different files generate the same checksum. In that case, Spdr may erroneously treat them as similar. To avoid this, you can enable Deep Search: it will make Spdr check all files using a both Md5 and Sha1 checksums, thus minimizing the possibility of a mistake</string>
             </property>
             <property name="text">
              <string>Perform deep search</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonSynchronize">
            <property name="toolTip">
             <string>Performs the synchronization when clicked. This may take a while</string>
            </property>
            <property name="text">
             <string>Synchronize!</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelSynchronizeResult">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabImport">
       <attribute name="title">
        <string>Import</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayoutImport">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Advanced options</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="labelCopyMode">
               <property name="text">
                <string>Copy mode</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QComboBox" name="comboBoxCopyMode">
               <property name="toolTip">
                <string>Use this control to specify whether the files from input folder should be copied to the output directory (&lt;b&gt;default&lt;/b&gt;) or moved (&lt;b&gt;Be careful!&lt;/b&gt; This option means that the file will be completely removed from input folder!).</string>
               </property>
               <item>
                <property name="text">
                 <string>Copy</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Move</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonImport">
            <property name="toolTip">
             <string>Performs the import operation when clicked</string>
            </property>
            <property name="text">
             <string>Import!</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QWidget" name="widgetBasicImport" native="true">
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>425</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelImportResult">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAboutSpdr"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionResetSettings"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuLanguage">
    <property name="toolTip">
     <string>Select application language from a list of available languages. Changes require application restart to take effect.</string>
    </property>
    <property name="title">
     <string>Language</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuLanguage"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAboutSpdr">
   <property name="text">
    <string>About Spdr</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionResetSettings">
   <property name="text">
    <string>Reset settings</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
